examples/ucontext-cp.c

```diff
@@ -60,23 +62,18 @@ DEFINE_AWAIT_OP(readv)
 DEFINE_AWAIT_OP(writev)
 #undef DEFINE_AWAIT_OP
 
-static int setup_context(unsigned entries, async_context *pctx)
+static int setup_context(async_context *pctx, struct io_uring *ring)
 {
 	int ret;
 
-	ret = io_uring_queue_init(entries, &pctx->ring, 0);
-	if (ret < 0) {
-		fprintf(stderr, "queue_init: %s\n", strerror(-ret));
-		return -1;
-	}
-
+	pctx->ring = ring;
 	ret = getcontext(&pctx->ctx_fnew);
 	if (ret < 0) {
 		perror("getcontext");
 		return -1;
 	}
 	pctx->ctx_fnew.uc_stack.ss_sp = &pctx->stack_buf;
-	pctx->ctx_fnew.uc_stack.ss_size = sizeof(pctx->stack_buf);
+	pctx->ctx_fnew.uc_stack.ss_size = sizeof pctx->stack_buf;
```

Comment:
Please don't do that, it's correct as it is. sizeof() should include parens. Same in a few other spots.